src/shared/theme/colors.ts

```diff
@@ -25,6 +25,22 @@ export default {
     50: '#E0E1FF',
   },
   success: '#1CCB00',
+  transparentSuccess: 'rgba(0, 219, 176, 0.08)',
   error: '#FF3861',
+  transparentError: 'rgba(255, 56, 97, 0.08)',
   warning: '#EB4F1E',
+  transparentWhite: {
+    100: 'rgba(255,255,255, 0.04)',
+    200: 'rgba(255,255,255, 0.06)',
+    300: 'rgba(255,255,255, 0.32)',
+  },
+  transparentBlack: {
+    100: 'rgba(0,0,0, 0.24)',
+    200: 'rgba(0,0,0, 0.54)',
+    300: 'rgba(0,0,0, 0.66)',
+  },
+  transparentPrimary: {
+    100: 'rgba(180, 187, 255, 0.06)',
+    200: 'rgba(180, 187, 255, 0.12)',
```

Comment:
I don't think `100` `200` and `300` are correct names here, that's what you call range of colors. Here let's maybe use the percentage of opacity as the key? Figma uses names like this:
<img width="219" alt="Screenshot 2021-03-17 at 16 21 35" src="https://user-images.githubusercontent.com/12646744/111492347-e08c2300-873c-11eb-940d-ad00407aa6cd.png">

So it would be the most straight-forward to use `colors.transparentBlack[54]` in the code